### ISSUE TITLE
Problem with a publication and the linked source entities in WissKI

### ISSUE THREAD

**Issue by martonrozsa88** [erc-releven/.github#41]
This issue is about a publication extracted from PBW. I noticed this problem among the prosopographical data of Eugenios, kouboukleisios, subordinate of krites Niketas Xiphilinos (Eugenios 104). The issue has the following details:

- in theory, our database uses two documents (nos. 1-2) from Actes de Xénophon, Archives de l’Athos XV, Paris 1986 (pp. 59-75 and 76-79, respectively) as publications
- both documents are from 1089, so both are relevant to our database
- only no, 1, pp. 59-75 is in WissKI, no. 2, pp. 76-79 is not there
- Eugenios is mentioned in the second document, some assertions have data from this missing publication
- the source entities linked to these assertions point to the right pages and lines but refer to the wrong publication (obviously because the right one is not recorded in the database)
- e. g. "**78.9** (Actes de Xénophon, Archives de l’Athos XV, Paris 1986, pp. **59-75**)"

My question is how we should handle this issue. I can correct the data manually in this case, but I want to ask you about it before I do anything.

**Comment by tla** [comment]
It seems that this problem isn't limited to Xenophontos; we will need to do some more serious debugging. Thanks for noticing this!

**Comment by tla** [comment]
I wrote a script to find out how big the problem is; it seems we have 290 factoids that shouldn't have been added, 128 that should have been and weren't, and 121 sources (affecting assertions from 1142 factoids) that should have been added and weren't. I'll address these problems one at a time.
[extraneous_factoids.txt](https://github.com/erc-releven/.github/files/15204169/extraneous_factoids.txt)
[omitted_factoids.txt](https://github.com/erc-releven/.github/files/15204170/omitted_factoids.txt)
[wrong_biblinks.csv](https://github.com/erc-releven/.github/files/15204171/wrong_biblinks.csv)

**Comment by martonrozsa88** [comment]
As far as I see, there is a lot of sources that are not added, I wonder if the list include those cases, when PBW refers to multiple places in a volume. For example, in the case of Eustathios 118, man of the despoina, PBW refers to 2.59.183, 72.186 in the 2nd volume of Βυζαντινὰ ἔγγραφα τῆς μονῆς Πάτμου (Patmos: Acts). In WissKI, this reference remained the same and is turned into one source entity, but it actually points to two separate documents. Therefore, the source should be split up into two.

**Comment by tla** [comment]
Yes - this is what I meant by "121 sources that should have been added and weren't." They are all cases of separate documents from the same publication, and in all of those cases, the assertion source references that should point to them point to a different document from that source instead.
